tpl/template_funcs.go

```diff
@@ -1132,6 +1134,35 @@ func highlight(in interface{}, lang, opts string) (template.HTML, error) {
 	return template.HTML(helpers.Highlight(html.UnescapeString(str), lang, opts)), nil
 }
 
+// execute a command and returns its output as string
```

Comment:
Use a proper godoc comment.  Start with the func name and use complete sentences with punctuation.

`// exec executes a command and returns its output as a string.`